### PR TITLE
[MIG] account_invoice_overdue_reminder: allow to remove some undesired invoices on the fly during reminder process

### PR DESCRIPTION
**Pull request by matthieusaison**

### CODE REVIEW

File: account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py [hunk 1]
```diff
@@ -410,6 +412,9 @@ def button_compute_mail_info(self):
 
             # Hide button
             step.display_button_recompute_mail_info = False
+        action = self.env.ref(MOD + ".overdue_reminder_step_onebyone_action").read()[0]
```
Comment (legalsylvain) [inline account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py:427]: what is "MOD" ?

Comment (matthieusaison) [reply account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py:427]: It's the name of the module:
MOD = "account_invoice_overdue_reminder"

File: account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py [hunk 2]
```diff
@@ -378,28 +377,49 @@ class OverdueReminderStep(models.TransientModel):
     def _reminder_type_selection(self):
         return self.env["overdue.reminder.action"]._reminder_type_selection()
 
+    @api.onchange("invoice_ids")
+    def onchange_invoice_ids(self):
```
Comment (sebastienbeau) [inline account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py]: onchange are deprecated use
-  display_button_recompute_mail_info : compute + readonly=False
- counter :  normal compute field

File: account_invoice_overdue_reminder/wizard/overdue_reminder_wizard_view.xml [hunk 3]
```diff
@@ -70,7 +70,7 @@
         <form>
             <group name="main" col="4">
                 <field name="commercial_partner_id" />
-                <field name="counter" />
+                <field name="counter" force_save="1" />
```
Comment (sebastienbeau) [inline account_invoice_overdue_reminder/wizard/overdue_reminder_wizard_view.xml]: force_save can be remove if counter is a computed field

**Comment by sebastienbeau** [comment]
close as this PR introduce the same change : https://github.com/OCA/credit-control/pull/282

**Pull request closed without merging**